certbot/plugins/disco.py

```diff
@@ -13,11 +13,11 @@
 from certbot import interfaces
 
 try:
-    from collections import OrderedDict
+    from collections import OrderedDict  # type: ignore
 except ImportError:  # pragma: no cover
     # OrderedDict was added in Python 2.7
-    from ordereddict import OrderedDict  # pylint: disable=import-error
-
+    from ordereddict import OrderedDict as _OrderedDict  # pylint: disable=import-error
+    OrderedDict = _OrderedDict  # type: ignore
```

Comment:
If putting both the `pylint: ...` and `type: ...` on the same line doesn't work for whatever reason (including excess line length), you should be able to just move the `pylint:` message to the line before `from ordereddict import OrderedDict`. This disables that warning for the entire block, but because the `import` whose error we want to ignore is the only statement in the block, this shouldn't be a problem.